rom/laghos.cpp

```diff
@@ -1230,7 +1230,6 @@ int main(int argc, char *argv[])
                             else
                                 outfile_twp << cutoff[0] << ", " << cutoff[1] << ", " << cutoff[2] << "\n";
                         }
-                        delete samplerLast;
```

Comment:
Why should this `delete` be deleted?

Reply:
Yes, `delete samplerLast` should be kept.